### ISSUE TITLE
Add setting to display Feerate as percentage

### ISSUE THREAD

**Issue by sendbitcoin** [Ride-The-Lightning/RTL#469]
Aa far as my understanding goes the feerate is just a percentage fee (correct me if i'm wrong) of the amount routed.

The current behavior of displaying it as a fixed amount is needlessly confusing and very annoying. (especially given the current wording of (milli mSats) and no tooltip explaining it in plain english)

Please add an option/setting to display the feerate as a percentage.

**Current behavior:**
Fee Rate (milli mSats)
1000

**Expected behavior:**
Fee Rate (%)
0.1%

This would make understanding the routing fees a channel has a lot easier for the average person and would improve the UX.

**Comment by cubiclestorm** [comment]
Last update was to add this to 0.10.0 but I see that's what we're at now. Do we have any projection on if this will be in the next version?

**Comment by saubyk** [comment]
This is handled now with the release v0.11.0
